connectivity/check/deployment.go

```diff
@@ -1206,10 +1197,8 @@ func (ct *ConnectivityTest) validateDeployment(ctx context.Context) error {
 		}
 	}
 
-	svcDNSCtx, svcDNSCancel := context.WithTimeout(ctx, ct.params.ipCacheTimeout())
-	defer svcDNSCancel()
 	for _, cp := range ct.clientPods {
-		err := ct.waitForServiceDNS(svcDNSCtx, cp)
+		err := WaitForCoreDNS(ctx, ct, cp)
```

Comment:
nit: one liner err check.

Reply:
I'd personally keep as it is as now to reduce the code churn, given that it would be functionally equivalent. I don't have a strong opinion though, let me know in case you prefer that I change it.

Reply:
Just a nit. Figured we should be consistent.

Reply:
Makes sense. Let me update it.